package.json

```diff
@@ -5,11 +5,11 @@
     "prebuild": "graph codegen",
     "build": "graph build --network ",
     "predeploy-test": "yarn run build goerli --network-file ./networks.json",
-    "deploy-test": "graph deploy --product hosted-service kibagateaux/dd-test",
-    "deploy": "graph deploy --node https://api.thegraph.com/deploy/ kibagateaux/dd-test",
-    "create-local": "graph create --node http://localhost:8020/ kibagateaux/dd-test",
-    "remove-local": "graph remove --node http://localhost:8020/ kibagateaux/dd-test",
-    "deploy-local": "graph deploy --node http://localhost:8020/ --ipfs http://localhost:5001 kibagateaux/debt-dao"
+    "deploy-test": "graph deploy --product hosted-service hepnerthomas/dd-test",
```

Comment:
We should either remove this from script to allow it passed in as a parameter or should use an shell environment variable

Reply:
Any particular reason why? These are defaults that come when you create a new subgraph with `graph init`.

Reply:
@kibagateaux These have been removed.

Reply:
@kibagateaux I added these npm scripts back in and parameterized the subgraph url by creating a `.env` file with a `SUBGRAPH_URL` parameter. See the newly added `README.md` for additional details.